src/main/java/seedu/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.duke.command;
+
+import seedu.duke.model.Shelf;
+
+public class ExitCommand extends Command {
+
+    private static final String BYE_MESSAGE = "See you next time";
+
+    /**
+     * Override superclass Command to not perform any actions.
+     *
+     * @param list the Shelf in which exit command is executed
+     */
+    public void execute(Shelf list) {
+        System.out.println(BYE_MESSAGE);
+    }
```

Comment:
Please change the return type of execute function of all method to String in the next PR so that Lixin can work on implementing UI package.